router/core/graphql_handler.go

```diff
@@ -380,6 +381,14 @@ func propagateSubgraphErrors(ctx *resolve.Context) {
 	addErrorToSpan(ctx.Context(), err)
 }
 
+func logSubgraphErrors(ctx *resolve.Context, requestLogger *zap.Logger) {
```

Comment:
I think we can do it inside `propagateSubgraphErrors` so we don't need an extra function here.

Reply:
Sure, no problem ! It is done.